browser/ui/BUILD.gn

```diff
@@ -252,15 +252,6 @@ source_set("ui") {
       ]
     }
 
-    if (enable_brave_translate_extension) {
```

Comment:
there is also a deps section with `enable_brave_translate_extension`

Reply:
Removed